### DISCUSSION TITLE
Naming for this product and repo

### DISCUSSION THREAD

**Discussion by aesedepece** [witnet/data-feeds-explorer#7] [General]
We may need to get on the same page on the naming of this product and repo, before it's too late and we get used to something not so catchy.

For those who don't know, this repo contains the source code for the front end and back end of a little website that will allow smart contract developers to discover different publicly available data feeds on multiple chains. Initially, these feeds may be operated by Witnet Foundation, but there's no reason for listing feeds created by others.

**Comment by aesedepece** [comment]
IMO, a first iteration of the name could be _"witnet data feeds explorer"_.

**Reply by mariocao** [reply]
Sounds good to me!
